src/style.css

```diff
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.cont {
+  display: flex;
+  max-width: 1199px;
```

Comment:
snake should be always full-width
<img width="1440" alt="Screenshot 2022-07-26 at 09 10 41" src="https://user-images.githubusercontent.com/94289038/180935935-bb80ca10-7994-49b2-ba8a-22da7451755b.png">